### PR TITLE
Lovlelace: update pic glance

### PR DESCRIPTION
**Pull request by c727**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_lovelace/picture-glance.markdown [hunk 1]
```diff
@@ -2,128 +2,96 @@
...
-  image: https://images.pexels.com/photos/276724/pexels-photo-276724.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=240&w=495
-  title: Living
+  title: Living room
   entities:
     - switch.decorative_lights
     - light.ceiling_lights
     - lock.front_door
     - binary_sensor.movement_backyard
     - binary_sensor.basement_floor_wet
+  image: /local/living_room.png
```
Comment (sjabby) [inline source/_lovelace/picture-glance.markdown]: Images should be uploaded to ‘/images/lovelace/’

File: source/_lovelace/picture-glance.markdown [hunk 2]
```diff
@@ -2,128 +2,96 @@
...
     - light.ceiling_lights
-  state_filter:
-    - 'on'
-  card: picture-glance
-  card_config:
-    title: Lights
-    image: https://images.pexels.com/photos/356048/pexels-photo-356048.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=295&w=490
+  state_image:
+    "on": /local/living_room_on.png
+    "off": /local/living_room_off.png
```
Comment (sjabby) [inline source/_lovelace/picture-glance.markdown]: Images should be uploaded to ‘/images/lovelace/’

File: source/_lovelace/picture-glance.markdown (hunk 1 above)
Comment (arsaboo) [reply source/_lovelace/picture-glance.markdown]: @sjabby Those are just placeholders. We don't necessarily need those images.

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**